ea.go

```diff
@@ -13,6 +18,9 @@ type fileFullEaInformation struct {
 	ValueLength     uint16
 }
 
+//sys getFileEA(handle syscall.Handle, iosb *ioStatusBlock, buf *uint8, bufLen uint32, returnSingleEntry bool, eaList uintptr, eaListLen uint32, eaIndex *uint32, restartScan bool) (status ntstatus) = ntdll.NtQueryEaFile
+//sys setFileEA(handle syscall.Handle, iosb *ioStatusBlock, buf *uint8, bufLen uint32) (status ntstatus) = ntdll.NtSetEaFile
+
```

Comment:
For new bindings we should probably prefer `windows.Handle`. I see you have `GetFileEAHandle` defined to take in a `windows.Handle` also so it'll save the unnecessary cast when calling into these